fastlane/Pluginfile

```diff
@@ -1,13 +1,11 @@
 # frozen_string_literal: true
 
-# Autogenerated by fastlane
-#
 # Ensure this file is checked in to source control!
 
 group :screenshots, optional: true do
-  gem 'rmagick', '~> 3.2.0'
+  gem 'rmagick', '~> 4.1'
 end
 
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 4.1'
+gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit', branch: 'fix/promo-screenshots'
```

Comment:
@rachelmcr I've now approved the https://github.com/wordpress-mobile/release-toolkit/pull/366 PR but will let you merge it yourself whenever you're ready, as merging it will have the side-effect of deleting that `fix/promo-screenshots` branch. Once you hit the green button on the other PR, you can replace this with `branch: 'trunk'` for now until we release a new `release-toolkit` version (which we hope to do likely around next sprint or so) 🙂